lib/checks/tables/td-headers-attr-evaluate.js

```diff
@@ -24,7 +25,7 @@ function tdHeadersAttrEvaluate(node) {
     let isSelf = false;
     let notOfTable = false;
 
-    if (!cell.hasAttribute('headers')) {
+    if (!cell.hasAttribute('headers') || !isVisibleForScreenreader(cell)) {
```

Comment:
This should be it's own pr since it's going to start passing nodes it previously failed but not be in the changelog. That our the pr title needs to be updated to reflect both changes